config/config.ts

```diff
@@ -305,8 +345,26 @@ const config: Configuration = {
                         // description: a short description of why the addon is incompatible
                         {
                             title: 'Mets747 FBW A380 Lights Mod',
-                            description: "It is required to remove this add-on before installing and using the A32NX. This add-on overrides A32NX components and may render the A32NX unusable."
+                            description: "It is recommended to remove this add-on before installing and using the A380X. This add-on overrides " +
```

Comment:
Is it intentional to change only this to recommended and leave the others as required?

Reply:
Yes - as this mod builder added a disclaimer to his page stating, to only use this mod on stable as it might brake the aircraft when the aircraft is updated. I thought this warranted recognition.